hicexplorer/hicFindTADs.py

```diff
@@ -791,7 +791,7 @@ def save_domains_and_boundaries(chrom, chr_start, chr_end, matrix, min_idx,
 
     mean_mat_all = matrix.mean(axis=1)
 
-    prefix = args.outPrefix + "_delta{}_pval{}".format(args.delta, args.pvalue)
```

Comment:
users can add it if they want. But by default if i re-run the program with new params i can't overwrite the old files